modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/RebalanceRaftGroupEventsListener.java

```diff
@@ -136,18 +135,15 @@ public void onLeaderElected(long term) {
                 try {
                     rebalanceAttempts.set(0);
 
-                    metaStorageMgr.get(pendingPartAssignmentsKey(partId))
```

Comment:
Why we need this changes?

Reply:
This was made to decouple futures of metastorage.get() and changePeersAsync, so we could catch only metastorage related logic, and fix it in the todo ticket.